lib/screens/player_screen.dart

```diff
@@ -173,3 +163,107 @@ class _PlayerScreenContent extends StatelessWidget {
     );
   }
 }
+
+enum PlayerHideable {
+  bigPlayButton(14, 14),
+  queueButton(0, 27),
+  progressSlider(0, 14),
+  twoLineTitle(0, 27),
+  codecInfo(0, 20),
+  loopShuffleButtons(96, 0);
+
+  final double width;
+  final double height;
+
+  const PlayerHideable(this.width, this.height);
+}
+
+class PlayerHideableController {
+  final verticalHideOrder = [
+    PlayerHideable.bigPlayButton,
+    PlayerHideable.queueButton,
+    PlayerHideable.codecInfo,
+    PlayerHideable.twoLineTitle,
+    PlayerHideable.progressSlider
+  ];
+
+  List<PlayerHideable> _visible = [];
+  Size? _target;
+
+  void setSizePortrait(Size size) {
+    _visible = List.from(PlayerHideable.values);
+
+    var maxPadding =
+        FinampSettingsHelper.finampSettings.playerScreenCoverMinimumPadding + 8;
```

Comment:
Why is this 8 added here? Afaik the padding is expressed as a percentage, so adding 8 would make this 800% ?

Reply:
It's stored as a percentage, not a decimal.  So this adds 8%.  Basically this allows up to 8% extra padding per side to show more player controls if needed.  But if when we hit that point we begin hiding controls instead of shrinking the album further.

Reply:
Right, I totally forgot I used percentages right-away xD  
Should be fine then!